pom.xml

```diff
@@ -106,13 +107,36 @@
             <artifactId>jackson-databind</artifactId>
             <version>2.7.4</version>
         </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.2.3</version>
```

Comment:
Do you want to have it as a mandatory dependency?
If it's just for logback util, there are ways to make it optional. But no strong opinion

Reply:
True, I can define this as optional dependency and do a soft-fail.

Reply:
Done, thanks @oleg-nenashev  for your feedback, good catch.